circuits/census.circom

```diff
@@ -1,40 +1,38 @@
-pragma circom 2.0.0;
+pragma circom 2.1.0;
 
 /*
-# credential.circom
-
 Circuit to check:
-- the prover is the owner of the secret key
-- zkCensusKey (hash of the user's secret key) belongs to the census
-	- the secret key is inside a Hash, which is inside the Merkletree with
-	  the CensusRoot (key=index), value=zkCensusKey)
-- H(secretKey, processID) == nullifier
+- the prover is the owner of the private key
+- keyHash (hash of the user's public key) belongs to the census
+		- the public key is generated based on the provided private key
+		- the public key is inside a hash, which is inside the Merkletree
+		  with the CensusRoot and siblings (key=keyHash, value=weight)
+- H(private key, processID) == nullifier
 	- to avoid proof reusability
 
 
-                       +----------+
-                       |          |
-PUB_censusRoot+------->+          |(key)<-----+PRI_index
-                       |          |
-                       | SMT      |            +----------+
-                       | Verifier |            |          |
-PRI_siblings+--------->+          |(value)<----+ Poseidon +<-----+--+PRI_secretKey
-                       |          |            |          |      |
-                       +----------+            +----------+      |
-                                                                 |
-                                     +----------+                |
-                      +----+         |          +<---------------+
+					   +----------+
+					   |          |
+PUB_censusRoot+------->+          |(value)<-----+PUB_weight
```

Comment:
seems to be displaced now: https://github.com/vocdoni/zk-franchise-proof-circuit/blob/feature/standard-census-circuit/circuits/census.circom#L16

Reply:
solved!